### ISSUE TITLE
[Accessibility] A11y_AzureDataStudio_ProxiesTab_ScreenReader: Unnecessarily column mark-up read for the table by NVDA .

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5624]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks) Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
NVDA Version: 2019.1
JAWS Version: 2019

**Additional Details** 
Tool Name: NVDA
MAS Reference:MAS1.3.1

**Repro Steps:**
1.Launch Visual Studio code desktop application. 
2. Connect to the Database -> Go to the SQL Agent ->proxy tab
3.Start Screen reader as NVDA.
4.Navigate to table and the blank section in the table..
5.Observe that Screen reader is unnecessarily announcing column for the table.

**Actual:**
Screen reader is unnecessarily announcing column for the table.
**Similar issue is observed for the below pages:**

 **1.Jobs:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->
 Navigate to jobs page.
**2.Operatortab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to Operator tab
**3. Alertstab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to Alerts tab.

**Same issue observed with JAWS.**

**Expected:**
Screen reader should not read that column which does not have any information present for the user.

**User Impact:**
User will not be able to understand why the screen reader reading that column which is unnecessarily present. As there is no information present in the column for the user, it may create confusion for the user.

**Recommendation:**
Please refer the following snippet link for fixes: https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[5624_[Accessibility] A11y_Azure Data Studio_ProxiesTab_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3216768/5624_.Accessibility.A11y_Azure.Data.Studio_ProxiesTab_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes
